superset-installer/etc/reflex-provisioner/roles/superset/data_model/tasks/main.yml

```diff
@@ -1,36 +1,23 @@
 ---
+- name: create {{ superset_owner_user }} owner in postgres for {{ superset_postgres_db }} database
+  postgresql_user:
+    name: "{{ superset_owner_user }}"
+    login_host: "{{ superset_postgres_access_ip }}"
+    login_password: "{{ superset_postgres_db_passwd }}"
+    password: "{{ superset_owner_password }}"
+    role_attr_flags: CREATEDB
+    port: "{{ superset_postgres_access_port }}"
+  become: yes  
+  become_user: postgres
```

Comment:
By setting  become : yes  we are using become_user : postgres  user  to run create db query  in PostgreSQL, if become: no then it will execute query with superuser